src/components/apps/dungeon/home.tsx

```diff
@@ -986,6 +986,7 @@ export function DungeonApp()
                     setEnemyState(DungeonStatus.alive);
                     
                     //player death audio
+                    PlayerDeathAudio.volume=volume/100
                     playAudio(PlayerDeathAudio)
```

Comment:
why not change playAudio to include a volume, that way you dont need this line above each audio element, and it won't be necessary to remember to do this for future sound effects